legend-engine-xts-java/legend-engine-xt-javaPlatformBinding-pure/src/main/resources/core_java_platform_binding/legendJavaPlatformBinding/planConventions/enginePlatformDependencies.pure

```diff
@@ -222,6 +222,10 @@ function meta::pure::executionPlan::platformBinding::legendJava::applyJavaEngine
       ->addMethod(javaMethod('public', $jPureDate, 'firstDayOfThisYear', []))
       ->addMethod(javaMethod('public', $jPureDate, 'firstDayOfWeek', [javaParam($jPureDate, 'p0')]))
       ->addMethod(javaMethod('public', $jPureDate, 'firstDayOfYear', [javaParam($jPureDate, 'p0')]))
+      ->addMethod(javaMethod('public', $jPureDate, 'firstHourOfDay', [javaParam($jPureDate, 'p0')]))
```

Comment:
Were these auto generated? It seems we are not registering all library functions we added?

Reply:
no they were not autogenerated